### ISSUE TITLE
Scale on iOS devices

### ISSUE THREAD

**Issue by falget** [kutlugsahin/ngx-smooth-dnd#2]
Hello, i found some bug, when i scale screen and start draging item his position is wrong. Item display not the same position where i touch. Reproduced only on iOS devices, on Android devices work correctly.

Sorry but i can't attach screenshot or video.

**Comment by kutlugsahin** [comment]
Hi @falget ,
I don't have ios device to test and debug right now but do you know if there is a ios safari specific event that is fired when you 'scale'. I assume you mean pinch zoom by scaling right? If so it may be enough to  handle **ongestureend** event  for ios

**Comment by falget** [comment]
Yes i mean pinch zoom, i tried to fix this problem by disable scaling on page, but on iOS i can't do this totally. To check scaling i tried to use next events:

> 	@HostListener('document:gesturestart', ['$event'])
	@HostListener('document:gesturechange', ['$event'])
	@HostListener('document:touchstart', ['$event'])
	@HostListener('document:touchmove', ['$event'])
        onScale(event: any) {
        event = event.originalEvent || event;
        if (event.scale !== 1) {
           event.preventDefault();
        }
    }

I'm not sure but i think, when dragging need to somehow calculate the position in a different way taking into account the zoom.

**Comment by kutlugsahin** [comment]
Thanks @falget! I need to invalidate the layout calculations after this specific event. Probably will be fixed in the next version.

**Comment by falget** [comment]
@kutlugsahin Ok, thank you, will be wait.
